src/contact.rs

```
@@ -469,56 +468,58 @@ impl Contact {
                     row_name
                 };
 
-                let query = sqlx::query(
-                    "UPDATE contacts SET name=?, addr=?, origin=?, authname=? WHERE id=?;",
-                )
-                .bind(&new_name)
-                .bind(if update_addr {
-                    addr.to_string()
-                } else {
-                    row_addr
-                })
-                .bind(if origin > row_origin {
-                    origin
-                } else {
-                    row_origin
-                })
-                .bind(if update_authname {
-                    name.to_string()
-                } else {
-                    row_authname
-                })
-                .bind(row_id);
-
-                context.sql.execute(query).await.ok();
+                context
+                    .sql
+                    .execute(
+                        "UPDATE contacts SET name=?, addr=?, origin=?, authname=? WHERE id=?;",
+                        paramsv![
+                            new_name,
+                            if update_addr {
+                                addr.to_string()
+                            } else {
+                                row_addr
+                            },
+                            if origin > row_origin {
+                                origin
+                            } else {
+                                row_origin
+                            },
+                            if update_authname {
+                                name.to_string()
+                            } else {
+                                row_authname
+                            },
+                            row_id
+                        ],
+                    )
+                    .await
+                    .ok();
```

Why we ignore the error here/